pkg/apis/core/validation/shoot_test.go

```diff
@@ -7063,6 +7044,32 @@ var _ = Describe("Shoot Validation Tests", func() {
 			))
 		})
 	})
+
+	Describe("#ValidateOIDCIssuerURL", func() {
+		DescribeTable("test  valid and invalid issuer URLs", func(issuerURL string, detail string) {
```

Comment:
Nit: I cannot unsee the extra whitespace 👀 
```suggestion
		DescribeTable("test valid and invalid issuer URLs", func(issuerURL string, detail string) {
```